datashuttle/tui/app.py

```diff
@@ -55,7 +55,13 @@ def compose(self) -> ComposeResult:
         )
 
     def on_mount(self) -> None:
-        self.dark = self.load_global_settings()["dark_mode"]
+        self.set_dark_mode(self.load_global_settings()["dark_mode"])
+
+    def set_dark_mode(self, dark_mode: bool) -> None:
+        if dark_mode:
+            self.theme = "textual-dark"
+        else:
+            self.theme = "textual-light"
```

Comment:
in case you want to be more concise
```suggestion
        self.theme = "textual-dark" if dark_mode else "textual-light"
```

Reply:
cheers!